diesel/src/row.rs

```diff
@@ -27,6 +27,12 @@ pub trait Row<DB: Backend> {
             self.take();
         }
     }
+
+    /// Number of columns in the current result set
+    fn column_count(&self) -> usize;
+
+    /// Name of the current column
```

Comment:
I think, now, because return type is `Option<...>`, it needs to clarify, in what situations `None` can be returned

Reply:
As Sean has stated above: A field in a select clause does not necessarily have a name.